modules/ROOT/pages/pricing-metrics.adoc

```diff
@@ -0,0 +1,138 @@
+= Usage Metrics
```

Comment:
MuleSoft captures usage data for Mule flows, Mule messages, and data throughput, but not all of this data is aggregated in usage reports.